web-console/4.0-designs/storage/storage.md

```diff
@@ -0,0 +1,22 @@
+# Storage
+
+## Create Storage Class
+
+![Storage class page with create button](img/create-storage-class.png)
+- Users with appropriate privileges can click to create a new Storage Class.
+
+![create storage class form](img/storage-class-form.png)
```

Comment:
Is there a possibility that the storage class fails to create?

Reply:
I'd expect that this would behave the same as any other failed resource creation. Do we have a set of standardized designs for those cases?

Reply:
Not sure right now, but I'm working on similar designs for those deploy image / create from YAML stories. Once we land on something it might be helpful to include here too!